packages/core/getBusyTimes.ts

```diff
@@ -39,12 +42,19 @@ export async function getBusyTimes(params: {
         startTime: true,
         endTime: true,
         title: true,
+        eventType: {
+          select: {
+            afterEventBuffer: true,
```

Comment:
we could add afterEventBuffer also as a parameter like beforeEventBuffer, to keep it consistent

Reply:
Not exactly - we need this event types afterEventBuffer not the one we are currently booking.